release_notes.md

```diff
@@ -0,0 +1,67 @@
+ # Release notes
+
+- Woof Mobile APP
+- Release notes version: 1
+- Release note date: May. 4, 2023
+
+All notable changes to this application will be documented in this file.
+
+## Overview
+
+This is the application for iOS.
```

Comment:
Why do we duplicating this info (I have seen it in the README already)?

Reply:
The duplicate information are removed.

Reply:
I think, that this files: release_notes.md and README.md is independent of each other. And users or developers can view only one of them. 
Now, for follow DRY principle the paragraph "Overview" is deleted.